tests/phpunit/includes/links-trait.php

```diff
@@ -35,7 +35,7 @@ function ( $url ) use ( $orig_siteurl ) {
 					$url = str_replace( $orig_siteurl, $siteurl, $url );
 				}
 
-				return str_replace( POLYLANG_DIR . '/', '/polylang/', $url );
+				return $url;
```

Comment:
Could you explain why you changed this line?

Reply:
A start of explanation in this commit https://github.com/polylang/polylang/commit/04ee8c1699d7bde4fb26dee8099dd905cb2a949d where I proposed to modify this part of code because this replacement doesn't work with Windows paths.

So when we pass through this code on Windows makes the replacement wrong in plugin URLs for 3 reasons:

- `plugins_url()` function normalizes paths See https://github.com/WordPress/WordPress/blob/6.6.1/wp-includes/link-template.php#L3654-L3656. So when we filter it Windows paths contains slash instead of backslash.
- `POLYLANG_DIR` isn't normalized at this point. So its value is can't be found in the normalized URL.

Example with Polylang flags, URL value at filter entry is
`http://example.org/subfolder/wp-content/plugins/C:/polylang/polylang/flags/us.png`

`C:\polylang\polylang/` can't be found and replaced because the value in URL is `C:/polylang/polylang/`.

So comparison in test fails.
`http://example.org/subfolder/wp-content/plugins/C:/polylang/polylang/flags/us.png` isn't equals to what we expect
`http://example.org/subfolder/wp-content/plugins/polylang/flags/us.png`

The third reason is that Windows paths never begin with a slash but with a drive letter. Even if we normalize `POLYLANG_DIR`, replace `POLYLANG_DIR . '/'` with `/polylang/` can't work because it produces a double slash.

`http://example.org/subfolder/wp-content/plugins/C:/polylang/polylang/flags/us.png` where we replace `C:/polylang/polylang/` with `/polylang/` makes `http://example.org/subfolder/wp-content/plugins//polylang/flags/us.png` which isn't equals `http://example.org/subfolder/wp-content/plugins/polylang/flags/us.png`

This modification and this other commit https://github.com/polylang/polylang/commit/7a70b5d865b0c8beb9b8c3d46407b621e668b6a7 led us to modify the PHPUnit bootstrap.

Then the solution developed in the PHPUnit bootstrap makes this replacement useless because it makes URL value at plugins_url filter entry correct.

It works because of the work of `plugin_basename()` and `plugins_url()` where `plugin_basename()` is called.
See 
- https://github.com/WordPress/WordPress/blob/6.6.1/wp-includes/link-template.php#L3667-L3670
- https://github.com/WordPress/WordPress/blob/master/wp-includes/plugin.php#L765-L786

`plugin_basename()` function uses `wp_plugin_paths` global variable.